luabind/adopt_policy.hpp

```diff
@@ -65,7 +65,11 @@ namespace luabind { namespace detail
 
             object_rep* obj = static_cast<object_rep*>(
                 lua_touserdata(L, index));
-            obj->release();
+
+            if (obj)
```

Comment:
Ideally, please add a unit test for this.

Reply:
Added unit test.